src/services/doppler-legacy-client.ts

```diff
@@ -410,6 +422,7 @@ export function mapHeaderDataJson(json: any) {
     },
     nav: (json.nav && json.nav.map(mapNavMainEntry)) || [],
     user: {
+      idUser: mapIdUserToken(json.jwtToken),
```

Comment:
```suggestion
      // TODO: Read directly the propertyy `idUser` after Doppler's PR #7335 is deployed to production  
      // See https://github.com/MakingSense/Doppler/pull/7335
      idUser: mapIdUserToken(json.jwtToken),
```

Reply:
It comment is obsolete because the PR in doppler was closed because the idUser is obtained from the token